remote/remote_state_s3.go

```diff
@@ -558,13 +562,15 @@ func checkIfS3BucketNeedsUpdate(s3Client *s3.S3, config *ExtendedRemoteStateConf
 		}
 	}
 
-	enabled, err := checkIfS3PublicAccessBlockingEnabled(s3Client, &config.remoteStateConfigS3, terragruntOptions)
-	if err != nil {
-		return false, configBucket, err
-	}
-	if !enabled {
-		configBucket.PublicAccess = true
-		needUpdate = append(needUpdate, "Bucket Public Access Blocking")
+	if !config.SkipBucketPublicAccessBlocking {
+		enabled, err := checkIfS3PublicAccessBlockingEnabled(s3Client, &config.remoteStateConfigS3, terragruntOptions)
+		if err != nil {
+			return false, configBucket, err
+		}
+		if !enabled {
+			configBucket.PublicAccess = true
+			needUpdate = append(needUpdate, "Bucket Public Access Blocking")
+		}
```

Comment:
I think this logic might be wrong?

Reply:
Hi, which part? as I understand from code, value `PublicAccess` is used later in the decision to invoke EnablePublicAccessBlockingForS3Bucket 

https://github.com/gruntwork-io/terragrunt/blob/master/remote/remote_state_s3.go#L481

Reply:
I just wanted to followup-- this logic is not wrong! It's just that the namings are misleading.